README.md

```diff
@@ -1,17 +1,77 @@
-1. Project info
-This project is about creating simple Tic Tac Toe game.
+
+TicTacToeGit
+
+Simple game of Tic Tac Toe 
+
+
+Getting Started
+
+/*These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.*/
+
 It suppose to:
 - enable  two players to have a game
 - display information whose turn is is
 - mark tiles with "X" or "0"
 - acknowledge end of the game by getting one of the winning combinations
 - display the result/winner
 
-2. Technology
+You can find a copy of the project here: https://github.com/Kesse8401/TicTacToe
+
+
+Prerequisites
+
+/* What things you need to install the software and how to install them */
+
+Live-server is required to start the project. You can find it here: https://www.npmjs.com/package/live-server along with the instructions regarding installing
+
+You can install live-server locally by navigating you your target file and adding this command to your terminal:
+
+npm install live-server --saveDev
+
+
+
+!!! Installing
+
+/* A step by step series of examples that tell you how to get a development env running */
```

Comment:
This is incomplete. No incomplete code/comment belongs on `master`

Reply:
Idk this one. Are those steps that I have completed? Or that somebody has to? Haven't we covered that already? (Download/clone , install, start) ?

Reply:
this bit:
```
Say what the step will be
+
+Give the example
+And repeat
```
seems to be a bit bullshit